packages/ndla-article-scripts/src/figureScripts.js

```diff
@@ -103,12 +103,14 @@ export const addEventListenerForFigureZoomButton = () => {
     const target = el;
     target.onclick = () => {
       const parentFigure = target.closest('figure');
+      const sourceTag = parentFigure.getElementsByTagName('source')[0];
```

Comment:
Gjør en sjekk på source for du henter ut [0]?

Reply:
Jepp!